lib/matplotlib/font_manager.py

```diff
@@ -86,6 +86,10 @@
     'extra bold': 800,
     'black':      900,
 }
+
+def map_weight_name_to_score(weight):
```

Comment:
this should be private

Reply:
When you say private, I think of other language access modifiers. In Python would that be just making this a class method?

Reply:
Just prepend an underscore to its name.